XmasPlugin/src/org/workcraft/plugins/xmas/tools/VerTool.java

```diff
@@ -403,13 +394,6 @@ public void run(final WorkspaceEntry we) {
                 str = processLoc(locFile.getAbsolutePath());
             }
             if (test > 0) {
-                if (display.equals("popup")) {
```

Comment:
Why? There are no side effects. And if there are any in the constructors, that's just lousy programming.